WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -31,32 +36,48 @@ public class PeopleViewController: UITableViewController, NSFetchedResultsContro
         return cell
     }
 
-    // Temporarily disable row selection until detail view is ready
-    override public func tableView(tableView: UITableView, willSelectRowAtIndexPath indexPath: NSIndexPath) -> NSIndexPath? {
-        return nil
-    }
-
+    
+    // MARK: - NSFetchedResultsController Methods
+    
     public func controllerDidChangeContent(controller: NSFetchedResultsController) {
         tableView.reloadData()
     }
 
+    
+    // MARK: - View Lifecycle Methods
+    
     public override func viewDidLoad() {
         super.viewDidLoad()
         do {
             try resultsController.performFetch()
         } catch {
             DDLogSwift.logError("Error fetching People: \(error)")
         }
+        
+        WPStyleGuide.configureColorsForView(view, andTableView: tableView)
     }
 
     public override func viewDidAppear(animated: Bool) {
         super.viewDidAppear(animated)
+// TODO: JLP May.3.2016. Spinner??
```

Comment:
I'd suggest to either:

1. Leave the TODO, but detail what needs to be done so that others can understand.  Ideally link a ticket since TODO == ticket.
2. Remove the TODO.